CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Added
+
+- Support for PHP version `8.0`
+
+### Changed
+
+- Updated environment (`Dockerfile`) for local development & tests (`php:7.4.29-alpine3.15`, `composer:2.3.5`, `xdebug-3.1.4`)
+
```

Comment:
- Package `"phpstan/phpstan" is fixed at `0.12.99`

Reply:
`CHANGELOG.md` updated